### PR TITLE
Updating repo with Microsoft Artifact Registry (also known as Microsoft Container Registry or MCR) information + terms of use

### PR DESCRIPTION
**Pull request by terencet-dev**
Signed-off-by: Terence Truong <48969648+terencet-dev@users.noreply.github.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,41 +1,57 @@
...
 ](https://cloudblogs.microsoft.com/opensource/2019/01/17/improved-discovery-experience-microsoft-containers-docker-hub/)
 
-## Browsing MCR Content
+## Browsing MAR Content
 
-The discovery experience for MCR is provided through [dockerhub](https://hub.docker.com/publishers/microsoftowner).
+The discovery experience for MAR is provided through [mcr.microsoft.com](https://mcr.microsoft.com/) and [Docker Hub](https://hub.docker.com/publishers/microsoftowner).
 
-To query the list of repositories within mcr: https://mcr.microsoft.com/v2/_catalog
+To query the list of repositories within MAR: https://mcr.microsoft.com/v2/_catalog
```
Comment (ritazh) [inline README.md:15]: Should we route users to the portal instead?

File: README.md [hunk 2]
```diff
@@ -1,41 +1,57 @@
...
 
-* Can I browse MCR? No, MCR does not have a UI/discovery experience, the only experience with the registry is for interacting with a container image. Ex: docker pull mcr.microsoft.com/windows/servercore:ltsc2019
+* **Will the endpoints change from mcr.microsoft.com to mar.microsoft.com?**
 
-* Can I Onboard new Images to MCR? No, Given that MCR is meant to host only official Microsoft Container Images, only Internal Microsoft Product teams will be able to onboard images to the registry. Once onboarded, the images are available to all public users.
+    No, all endpoints will use the existing `mcr.microsoft.com` structure.
+
+* **How does MAR work with Docker Hub?**  
+
+    Docker Hub is one of the official sources for our customers to discover official Microsoft-published container images. For further details of this integration please visit our [blog](https://cloudblogs.microsoft.com/opensource/2019/01/17/improved-discovery-experience-microsoft-containers-docker-hub/).
```
Comment (ritazh) [inline README.md:42]:
```suggestion
    MAR is a public registry for housing Microsoft's official artifacts (such as container images). Docker Hub is one of the official sources (in addition to https://mcr.microsoft.com) for our customers to discover official Microsoft-published container images. For further details of this integration please visit our [blog](https://cloudblogs.microsoft.com/opensource/2019/01/17/improved-discovery-experience-microsoft-containers-docker-hub/).

```

File: README.md (hunk 1 above)
Comment (terencet-dev) [reply README.md:15]: That's what the line 13 hopes to provide. Providing users the opportunity to browse

By keeping this line, this allows customers to query the repos without the portal. Thoughts?

File: README.md [hunk 3]
```diff
@@ -1,41 +1,57 @@
...
 
 - [Microsoft syndicates container catalog (mcr.microsoft.com)](https://azure.microsoft.com/blog/microsoft-syndicates-container-catalog/)
 - [Improved discovery experience for Microsoft containers on Docker Hub
 ](https://cloudblogs.microsoft.com/opensource/2019/01/17/improved-discovery-experience-microsoft-containers-docker-hub/)
 
-## Browsing MCR Content
+## Browsing MAR Content
 
-The discovery experience for MCR is provided through [dockerhub](https://hub.docker.com/publishers/microsoftowner).
+The discovery experience for MAR is provided through [mcr.microsoft.com](https://mcr.microsoft.com/) and [Docker Hub](https://hub.docker.com/publishers/microsoftowner).
```
Comment (ritazh) [inline README.md:13]:
```suggestion
The discovery experience for MAR is provided through the MAR portal [mcr.microsoft.com](https://mcr.microsoft.com/) and [Docker Hub](https://hub.docker.com/publishers/microsoftowner).

```

you refer to it as `MAR portal` below.

File: client-firewall-rules.md [hunk 4]
```diff
@@ -1,21 +1,21 @@
...
+Microsoft Artifact Registry (also known as Microsoft Container Registry or MCR) is an implementation of the [OCI Distribution Specification][oci-spec] which delivers [artifacts][oci-artifacts], such as container images, helm charts and other assets distributed by Microsoft and Azure. The Distribution Spec defines two endpoints:
 
 - **REST Endpoint**: providing content discovery. This is the url users are most familiar with when pulling an image: `docker pull mcr.microsoft.com/windows/servercore:1909`. The REST endpoint is load balanced across multiple worldwide regions, providing consistent content addressable artifacts.
 - **Data Endpoint**: providing content delivery. As a registry client discovers the content required, it negotiates a set of content urls, pulling from the data endpoint. To provide reliable delivery of content, the data endpoint is backed by regionalized CDN endpoints.
 
 ## Configuring Client Firewall Rules
 
-To access MCR, the following FQDNs are required.
+To access <AR, the following FQDNs are required.
```
Comment (ritazh) [inline client-firewall-rules.md:10]:
```suggestion
To access MAR, the following FQDNs are required.
```

File: README.md [hunk 5]
```diff
@@ -1,41 +1,62 @@
...
 
 - [Microsoft syndicates container catalog (mcr.microsoft.com)](https://azure.microsoft.com/blog/microsoft-syndicates-container-catalog/)
 - [Improved discovery experience for Microsoft containers on Docker Hub
 ](https://cloudblogs.microsoft.com/opensource/2019/01/17/improved-discovery-experience-microsoft-containers-docker-hub/)
 
-## Browsing MCR Content
+## Browsing MAR Content
 
-The discovery experience for MCR is provided through [dockerhub](https://hub.docker.com/publishers/microsoftowner).
+The discovery experience for MAR is provided through the MAR portal on [mcr.microsoft.com](https://mcr.microsoft.com/) and [Docker Hub](https://hub.docker.com/publishers/microsoftowner).
```
Comment (ritazh) [inline README.md:13]:
```suggestion
The discovery experience for MAR is provided through the MAR discovery UI on [mcr.microsoft.com](https://mcr.microsoft.com/) and [Docker Hub](https://hub.docker.com/publishers/microsoftowner).

```
Per Toddy's [comment](https://github.com/microsoft/containerregistry/pull/118#discussion_r881976863)

File: README.md [hunk 6]
```diff
@@ -1,41 +1,62 @@
...
+    MAR is a public registry for housing Microsoft's official artifacts (such as container images). Docker Hub is one of the official sources (in addition to https://mcr.microsoft.com) for our customers to discover official Microsoft-published container images. For further details of this integration please visit our [blog](https://cloudblogs.microsoft.com/opensource/2019/01/17/improved-discovery-experience-microsoft-containers-docker-hub/).
+
+* **What is the difference between MAR and ACR (Azure Container Registry)?**  
+
+    MAR is a public registry for housing only Microsoft's official artifacts (such as container images). ACR is a private container registry for housing our customers' container images.
+
+* **Can I browse MAR?**
 
-* What is the difference between MCR and ACR(Azure Container Registry)?  MCR is a public registry for housing only Microsoft's official Container images. ACR is a private container registry for housing our customers' container images.
+    Yes, customers can browse the MAR portal at [mcr.microsoft.com](https://mcr.microsoft.com/). Customers can view rich content like descriptions and tag listings for each artifact.
```
Comment (ritazh) [inline README.md:55]: ditto

File: README.md [hunk 7]
```diff
@@ -1,41 +1,62 @@
...
 ](https://cloudblogs.microsoft.com/opensource/2019/01/17/improved-discovery-experience-microsoft-containers-docker-hub/)
 
-## Browsing MCR Content
+## Browsing MAR Content
 
-The discovery experience for MCR is provided through [dockerhub](https://hub.docker.com/publishers/microsoftowner).
+The discovery experience for MAR is provided through the MAR discovery UI on [mcr.microsoft.com](https://mcr.microsoft.com/) and [Docker Hub](https://hub.docker.com/publishers/microsoftowner).
 
-To query the list of repositories within mcr: https://mcr.microsoft.com/v2/_catalog
+1. To query the list of repositories within MAR, visit https://mcr.microsoft.com/
```
Comment (AndreHamilton-MSFT) [inline README.md:15]: "To browse the repositories within MAR"?

File: README.md [hunk 8]
```diff
@@ -1,41 +1,62 @@
...
 
-For more background on MCR:
+For more background on MAR:
 
 - [Microsoft syndicates container catalog (mcr.microsoft.com)](https://azure.microsoft.com/blog/microsoft-syndicates-container-catalog/)
 - [Improved discovery experience for Microsoft containers on Docker Hub
 ](https://cloudblogs.microsoft.com/opensource/2019/01/17/improved-discovery-experience-microsoft-containers-docker-hub/)
 
-## Browsing MCR Content
+## Browsing MAR Content
```
Comment (AndreHamilton-MSFT) [inline README.md:11]: Should we have a separate section that maintains the information about programmatically querying the data using the tags and catalog endpoints?

Comment (terencet-dev) [reply README.md:11]: From the previous comments, from my understanding, we want users to query results from the portal

Comment (AndreHamilton-MSFT) [reply README.md:11]: Ah, missed those ones

**Pull request merged**